README.md

```diff
@@ -58,7 +58,11 @@ Clone this repository to your desired folder:
 
 Open your terminal:
 
-Run: `rails c`
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add the `bundle install` command in the Getting started section to make your readme more useful and descriptive 😊